src/main.js

```diff
@@ -50,6 +54,18 @@ process.on('unhandledRejection', err => {
   }
 })
 
+var shouldQuit = app.makeSingleInstance(function(commandLine, workingDirectory) {
+  // Someone tried to run a second instance, we should focus our window.
+  if (mainWindow) {
```

Comment:
I thought we weren't going to add this feature, as it prevents running multiple instances on different ports?

Reply:
I'm okay to leave it in, I guess - but we'll need to make sure it's highlighted in the release notes. I'm sure it'll piss some users off.

Reply:
We cannot currently run multiple instances on different ports because you currently can't run Ganache on multiple ports without a "hack" since both instances use the came config file.

The current "run on multiple ports" is due to a programming gap of checking things appropriately and I'd rather the application not fail for some users rather than let the hackers have their feature until we support it officially.

Reply:
Yeah, I'd err the other way, but I don't feel particularly strongly about it, so :+1: by me.